cmd/machine-api-operator/start.go

```diff
@@ -57,7 +57,9 @@ func init() {
 }
 
 func runStartCmd(cmd *cobra.Command, args []string) {
```

Comment:
If you change the signature here to return an error, and change `Run: runStartCmd,` to `RunE: runStartCmd` does that make the error handling easier? Means you don't have to use klog.Fatal and panics